scripts/src/java/org/oppia/android/scripts/todo/TodoCheck.kt

```diff
@@ -0,0 +1,140 @@
+package org.oppia.android.scripts.todo
+
+import com.squareup.moshi.JsonAdapter
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.Types
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import org.oppia.android.scripts.todo.data.Issue
+import java.io.File
+
+/**
+ * Script for ensuring that all TODOs present in the repository are correctly formatted and
+ * corresponds to an open issue on Github.
+ *
+ * Usage:
+ *   bazel run //scripts:todo_check -- <path_to_directory_root>
+ *
+ * Arguments:
+ * - path_to_directory_root: directory path to the root of the Oppia Android repository.
+ *
+ * Example:
+ *   bazel run //scripts:todo_check -- $(pwd)
+ *
+ * NOTE TO DEVELOPERS: The script is meant only to be run during the CI, as this expects a JSON
```

Comment:
I suggest passing in the JSON file path directly & updating this KDoc accordingly.

Reply:
Done.